align/schema/constraint.py

```diff
@@ -816,151 +816,35 @@ class DoNotUseLib(SoftConstraint):
     libraries: List[str]
     propagate: Optional[bool]
 
-
-class IsDigital(SoftConstraint):
-    '''
-    Place this hierarchy as a digital hierarchy
-    Forbids any preprocessing, auto-annotation,
-    array-identification or auto-constraint generation
-
-    Args:
-        isTrue (bool): true/false.
-        propagate: Copy this constraint to sub-hierarchies
-
-    Example: ::
-
-        {
-            "constraint": "IsDigital",
-            "isTrue": true,
-            "propagate": False
-        }
+class CompilerOpt(SoftConstraint):
```

Comment:
Rename CompilerOpt to ConfigureCompiler?

Reply:
Okay.

Reply:
Simply Compile is also an option.